frontend/src/components/ReviewSuccess.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "../css/Signup.css";
+
+function ReviewSuccess(props) {
+  return (
+    <div className="signup-form">
+      <div id="title-signup">
+        <b
+          className="text text-center "
+          style={{ fontSize: 40, color: "#2C6674" }}
+        >
+          Your review is added!
+        </b>
+      </div>
+      <p style={{ marginTop: 20 }}>
+        Back to <Link to="/">home page</Link>
```

Comment:
This ReviewSuccess page would be really helpful! My suggestion would be maybe to have a link that also redirects the user back to the HikePage they were on, if possible. Other than that thanks for adding this 👍

Reply:
Yes, I tried to do that but didn't know how to pass the hike id as props in here: https://github.com/MaggieYang57/CSC308-TeamB/blob/2c972d742fb4b00cdc6694483b19cf5dde3820b6/frontend/src/ReviewPage.js#L70

Maybe we can fix this in a follow up pull.